src/modules/flyout/flyout.module.ts

```diff
@@ -1,9 +1,21 @@
-import { SkyFlyoutComponent } from './flyout.component';
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
-import { SkyResourcesModule } from '../resources';
-import { SkyWindowRefService } from '../window';
+import {
+  NgModule
```

Comment:
Why do you break this instead of putting it on one line like some of the other one class import? (for my own knowledge :) )

Reply:
I always try to format barrel imports as multiline. Since the `from` source is pointing to a barrel, I write it with multiple lines (even if it's a single file) to make it easier for future developers to add other imports from the same barrel. 

Also, if we write it as multiline, future contributors will naturally write their imports in the same way. For example, if someone wanted to include `Component`, they would probably write the following, instead of adding it to a new line.

Not great:

```
import { NgModel, Component } from '@angular/core';
```

But if they're presented with this snippet, they're more likely to keep with the intended style:

```
import {
  NgModel
} from '@angular/core';
```